src/components/external-link/external-link.tsx

```diff
@@ -22,8 +22,19 @@ const ExternalLink = (props: Props) => {
     }
   }, [href, props.href]);
 
+  const handleEvent = (event: KeyboardEvent<HTMLAnchorElement> | MouseEvent<HTMLAnchorElement>) => {
```

Comment:
We don't want to do this globally. You've got a rather weird use case (a link inside an interactive element), so it should be handled there rather.

Reply:
I see. Done.
I had to extend ExternalLink to accept the anchor props and also took this opportunity to clean the `href` parsing logic a bit